### DISCUSSION TITLE
Label enforcement via PR check

### DISCUSSION THREAD

**Discussion by zackkrida** [WordPress/openverse#58] [Ideas]
How would folks feel about a PR check to ensure that `goal` and `aspect` labels are enforced on all pull requests? It sounds potentially annoying, but also it would be really nice to ensure PRs are labeled. My feelings are mixed.

**Comment by dhruvkb** [comment]
I'm in favour of having this (and all other labelling related automations). Properly labelled issues and PRs are good indicators of repository health and a very helpful tool for both new contributors as well as changelog generators.

It'll only be annoying till it becomes a habit.

**Comment by obulat** [comment]
I am all for this suggestion. Sometimes I don't check the labels because I'm not sure what exactly I need to check for, so having this guidance will be great!

**Comment by krysal** [comment]
I am in favor too, if the requirement is only for two labels is not a big burden and helps classify issues/PRs.

**Comment by zackkrida** [comment]
This has existed for some time now.
